docs/content/install/shared-team-instance.md

```diff
@@ -0,0 +1,174 @@
+The 🦁 says rawr!
+```
+
+
+**PowerShell**
+```console
+$env:GO111MODULE = "on"
+$env:GOPROXY = "http://127.0.0.1:3000"
+git clone https://github.com/athens-artifacts/walkthrough.git
+cd walkthrough
+$ go run .
+go: downloading github.com/athens-artifacts/samplelib v1.0.0
+The 🦁 says rawr!
+```
+
+We can verify that Athens handled this request by examining the Docker logs:
+
+```console
+$ docker logs -f athens-proxy
+time="2018-08-21T17:28:53Z" level=warning msg="Unless you set SESSION_SECRET env variable, your session storage is not protected!"
+time="2018-08-21T17:28:53Z" level=info msg="Starting application at 0.0.0.0:3000"
+handler: GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.info [200]
+handler: GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.mod [200]
+handler: GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.zip [200]
+```
+
+Now, if you view the contents of the **ATHENS_STORAGE** directory, you will see that you now have additional files representing the samplelib module.
+
+**Bash**
+```console
+$ ls -lr $ATHENS_STORAGE/github.com/athens-artifacts/samplelib/v1.0.0/
+total 24
+-rwxr-xr-x  1 jeremyrickard  wheel    50 Aug 21 10:52 v1.0.0.info
+-rwxr-xr-x  1 jeremyrickard  wheel  2391 Aug 21 10:52 source.zip
+-rwxr-xr-x  1 jeremyrickard  wheel    45 Aug 21 10:52 go.mod
+```
+
+**PowerShell**
+```console
+$ dir $ATHENS_STORAGE/github.com/athens-artifacts/samplelib/v1.0.0/
+total 24
+-rwxr-xr-x  1 jeremyrickard  wheel    50 Aug 21 10:52 v1.0.0.info
+-rwxr-xr-x  1 jeremyrickard  wheel  2391 Aug 21 10:52 source.zip
+-rwxr-xr-x  1 jeremyrickard  wheel    45 Aug 21 10:52 go.mod
+```
+
+When Athens is restarted, it will serve the module from this location without re-downloading it. To verify that, we need to first remove the Athens container.
+
+```console
+docker rm -f athens-proxy
+```
+
+Now, we need to clear the local Go modules cache. This is needed so that your local Go command line tool will re-download the module from Athens. The following commands will clear the local module cache:
+
+**Bash**
+```bash
+sudo rm -fr "$(go env GOPATH)/pkg/mod"
+```
+
+**PowerShell**
+```powershell
+rm -recurse -force $(go env GOPATH)\pkg\mod
+```
+
+Now, we can re-run the Athens container:
+
+**Bash**
+```console
+docker run -d -v $ATHENS_STORAGE:/var/lib/athens \
+   -e ATHENS_DISK_STORAGE_ROOT=/var/lib/athens \
+   -e ATHENS_STORAGE_TYPE=disk \
+   --name athens-proxy \
+   --restart always \
+   -p 3000:3000 \
+   jeremyrickard/athens-proxy:latest
+```
+
+**PowerShell**
+```console
+docker run -d -v $env:ATHENS_STORAGE:/var/lib/athens `
+   -e ATHENS_DISK_STORAGE_ROOT=/var/lib/athens `
+   -e ATHENS_STORAGE_TYPE=disk `
+   --name athens-proxy `
+   --restart always `
+   -p 3000:3000 `
+   jeremyrickard/athens-proxy:latest
+```
+
+When we re-run our Go example, the Go cli will again download module from Athens. Athens, however, will not need to retrieve the module. It will be served from the existing storage.
```

Comment:
suggestion: "It will be served from the Athens on-disk storage" or something like that.